airsonic-main/src/main/java/org/airsonic/player/dao/GenreDao.java

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
Because we are adding here a new table/DAO/domain object, we should start to write unit and integration tests also.